src/main/java/io/confluent/idesidecar/restapi/messageviewer/strategy/ConfluentCloudConsumeStrategy.java

```diff
@@ -222,6 +225,34 @@ private PartitionConsumeData processPartition(
     );
   }
 
+  private static List<SimpleConsumeMultiPartitionResponse.PartitionConsumeRecordHeader>
```

Comment:
nit: I don't think this method needs to be `static`. OTOH, it probably doesn't hurt.

Reply:
Ah makes sense. I frequently use the `Extract to method` feature in IntelliJ and didn't pay attention to whether it was static or not.